boreas/alivedetection.c

```diff
@@ -97,9 +97,11 @@ scan (alive_test_t alive_test)
       scanner.tcp_flag = TH_ACK;
       g_hash_table_foreach (scanner.hosts_data->targethosts, send_tcp,
                             &scanner);
+      usleep (500000);
```

Comment:
In gvmd we have [gvm_usleep](https://github.com/greenbone/gvmd/blob/master/src/utils.c#L61).  It does the same, but checks for interrupts from signals.  Not sure there's anything to gain in using it, but something to consider.